portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,145 @@
+:root {
+    --target-color: red; 
+    --circle-cursor-color: black; 
+    --highlighted-text-color: white;
+    --extra-content-opacity: 0;  
+}
+
+html, body {
+    cursor: none; 
+    margin: 0;
+    height: 100%;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+#images-parent {
+  position: relative;
+  top: 0;
+  left: 0;
+}
+#me-pic {
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 15rem;
+  height: 15rem; 
+  border-radius: 50%;
+  z-index: 1; 
+  /* border: 1px red solid; */
+}
+#tomie-pic {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 15rem;
+  height: 15rem; 
+  border-radius: 50%;
+  z-index: -1; 
+  /* border: 1px green solid; */
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+
+#icon {
+    width: 100px;
+    height: 50px;
+    background: aqua; 
+}
+
+.cursor {
+    width: 2rem; 
+    height: 2rem;
+    border: 2px solid var(--circle-cursor-color); 
+    border-radius: 50%; 
+    position: absolute; 
+    transform: translate(-50%,-50%); 
+    pointer-events: none; 
+    transition: all 0.3s ease; 
+    transition-property: background, transform;
+    transform-origin: 100% 100%; 
+    z-index: -1;  
+}
+
+.sections {
+    padding: 1rem; 
+    width: 80%; 
+    display: flex; 
+    justify-content: space-between;
+    list-style:none; 
+}
+
+.grow-cursor {
+    transform: scale(2); 
+    background: var(--circle-cursor-color); 
+}
+
+.color-text {
+    color: var(--highlighted-text-color);  
+}
+
+.main-text {
+    color: rgba(0, 0, 0, 0); 
+}
+
+.fade-text-in {
+    color: rgba(0, 0, 0, 1);
+    transition: all 1s ease; 
+}
+
+.fade-image-out {
+    opacity: 0;
+    transition: all 5s ease; 
+}
+
+.eerie-text{
+    /* color: #b00f00; */
+    color: var(--target-color); 
+    transition: all 2s ease; 
+}
+
+#extra-content {
+ /* TODO: FIGURE OUT WHY Z-INDEX DOESN'T SEEM TO BE CHANGING THE DEPTH */
+  z-index: -1; 
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px; 
+  opacity: 0; 
+}
+
+.change-extra-content-opacity {
```

Comment:
this is a verb which should only be for a function name, a class name would be an adjective/noun like "more-opaque-extra-content"